lib/config/types.ts

```diff
@@ -99,7 +99,8 @@ export interface RepoGlobalConfig {
   dryRun?: boolean;
   exposeAllEnv?: boolean;
   migratePresets?: Record<string, string>;
-  privateKey?: string | Buffer;
+  privateKey?: string;
+  privateKeyOld?: string;
```

Comment:
```suggestion
  privateKey?: string | Buffer;
  privateKeyOld?: string | Buffer;
```

I think this can be buffer, see:
https://github.com/renovatebot/renovate/blob/11694e92949171da339e54fe0d31b1f69532a6ad/lib/workers/global/config/parse/index.ts#L41-L44

Reply:
In that case can we convert from Buffer to string here instead?

Reply:
not sure, it depend on the key format, can be binary 🤷‍♂️

Reply:
We assume it's an armored key, I will fix in the read